### PR TITLE
Clean inappropriate code

### PR DESCRIPTION
**Pull request by crypto-matto**

### CODE REVIEW

File: src/pages/staking/staking.tsx [hunk 1]
```diff
@@ -595,8 +589,8 @@ const FormWithdrawStakingReward = () => {
   // TO-DO temporary CRO Asset select
   const assetType =
     currentSession.wallet.config.name === DefaultWalletConfigs.TestNetConfig.name
-      ? UserAssetType.TENDERMINT
-      : UserAssetType.TENDERMINT;
+      ? DefaultWalletConfigs.TestNetConfig.assetIdentifer?.cro
+      : DefaultWalletConfigs.MainNetConfig.assetIdentifer?.cro;
   const croAsset = TransactionUtils.getAssetFromAllAssets(walletAllAssets, assetType);
   setWalletAsset(croAsset);
```
Comment (crypto-3dd) [inline src/pages/staking/staking.tsx:595]: This is still not very scalable, you will need to remove entirely `TransactionUtils.getAssetFromAllAssets`, it's not a good pattern if we want to scale to multiple assets.

File: src/config/StaticConfig.ts [hunk 2]
```diff
@@ -46,6 +46,10 @@ export type WalletConfig = {
     gasLimit: string;
     networkFee: string;
   };
+  assetIdentifer?: {
+    cro: string;
+    cronos: string;
+  };
```
Comment (crypto-3dd) [inline src/config/StaticConfig.ts:52]: This doesn't scale very well as assets count, we can't hard code `assetID`, would you add this for every asset we add in the future.

I still don't know why we would need to hard code this, can you please elaborate.

File: src/config/StaticConfig.ts [hunk 3]
```diff
@@ -156,6 +160,10 @@ const MainNetConfig: WalletConfig = {
     gasLimit: FIXED_DEFAULT_GAS_LIMIT,
     networkFee: FIXED_DEFAULT_FEE,
   },
+  assetIdentifer: {
+    cro: 'e9cc453a9e39718c',
+    cronos: 'd1a17aca33fd382c',
+  },
```
Comment (crypto-3dd) [inline src/config/StaticConfig.ts:166]: We can't hardcode these values, they are created on the fly and shouldn't checked against.

Comment (crypto-matto) [reply src/config/StaticConfig.ts:166]: I see. I thought the id of cro & cronos are always the same. I misunderstood it.

File: src/pages/staking/staking.tsx (hunk 1 above)
Comment (crypto-matto) [reply src/pages/staking/staking.tsx:595]: Right now CRONOS is set as the only non-secondary asset. Yet I still have to pick CRO from allAsset to support CRO-only delegation related operations for 1st release. That’s why I need to use a function like `getAssetFromAllAssets()` (Temporary) to get me CRO from allAsset , and I used `identifier` & `assetType` (which I misunderstood its uniqueness).

**Pull request merged**